package.json

```diff
@@ -8,13 +8,16 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "lint": "eslint ./",
-    "format:verify": "prettier --list-different '{./components,./pages,./__tests__,./docs}/**/*.js'",
-    "format": "prettier --write '{./components,./pages,./__tests__,./docs}/**/*.js'"
+    "format:verify":
```

Comment:
We should bump the version number of the package.json as this is the first additions of new features to our app